keepAnimals

```diff
@@ -0,0 +1,7 @@
+def keepAnimals(animal_dic):
+  keepAnimals_dic = {}
+  for key, value in animal_dic.items():
```

Comment:
Can you rename `key` to `word` and `value` to `count`? This makes the code easier to understand.